src/server/pkg/hashtree/hashtree.go

```diff
@@ -596,7 +596,7 @@ func (h *hashtree) DeleteFile(path string) error {
 		return errorf(Internal, "delete discovered orphaned file \"%s\"", path)
 	}
 	if node.DirNode == nil {
-		return errorf(Internal, "node at \"%s\" is a file, but \"%s\" exists "+
+		return errorf(Internal, "file at \"%s\" is a regular-file, but \"%s\" exists "+
```

Comment:
Not sure if this is quite right, but something like `file at \"%s\" is a regular-file, but the \"%s\" already exists` would be even clearer

Reply:
oh that "the" was a typo by me. Sorry.